app/src/main/java/io/pslab/widgets/loggedData/logDataWidgetService.java

```diff
@@ -0,0 +1,13 @@
+package io.pslab.widgets.loggedData;
+
+import android.content.Intent;
+import android.widget.RemoteViewsService;
+
+public class logDataWidgetService extends RemoteViewsService {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention).
Class names should be nouns, in mixed case with the first letter of each internal word capitalized. Try to keep your class names simple and descriptive. Use whole words-avoid acronyms and abbreviations (unless the abbreviation is much more widely used than the long form, such as URL or HTML).

Reply:
Okay I will commit the changes